CHANGELOG.md

```diff
@@ -117,6 +117,7 @@
 - Display a clearer error message when `Index.names` is set to a non-like-like object.
 - Raise a warning whenever MultiIndex values are pulled in locally.
 - Improve warning message for `pd.read_snowflake` include the creation reason when temp table creation is triggered.
+- Improved readability for binary operation utility code.
```

Comment:
I thought this changelog was only meant for users. If so, we should delete this.

```suggestion
```

Reply:
I used to think the same as you but I changed:)
I feel as an open source library, our users are also developers, so it does not hurt to give more description for each change. 
Also, if unfortunately some regression happens, everyone can be easier to track to potential issues based on the log here.

Reply:
I also don't think this needs a CHANGELOG but there isn't any downside to having an entry here.

Reply:
This is okay with me.

Reply:
I'll remove this for now. I think the point is that probably no much value to deliver this message to the user.

Reply:
Yes, including all refactoring work here can add a lot of unnecessary clutter to the changelog.